llvm/test/CodeGen/AArch64/ptrauth-bti-call.ll

```diff
@@ -1,101 +1,128 @@
-; RUN: llc -mtriple arm64e-apple-darwin -mattr=+bti              -asm-verbose=false -o - %s | FileCheck %s
-; RUN: llc -mtriple arm64e-apple-darwin -mattr=+bti -global-isel -asm-verbose=false -o - %s | FileCheck %s
-; RUN: llc -mtriple arm64e-apple-darwin -mattr=+bti -fast-isel   -asm-verbose=false -o - %s | FileCheck %s
+; RUN: llc -mtriple arm64e-apple-darwin -mattr=+bti              -asm-verbose=false \
+; RUN:   -o - %s | FileCheck %s --check-prefixes=DARWIN,CHECK
+; RUN: llc -mtriple arm64e-apple-darwin -mattr=+bti -global-isel -asm-verbose=false \
+; RUN:   -o - %s | FileCheck %s --check-prefixes=DARWIN,CHECK
+; RUN: llc -mtriple arm64e-apple-darwin -mattr=+bti -fast-isel   -asm-verbose=false \
+; RUN:   -o - %s | FileCheck %s --check-prefixes=DARWIN,CHECK
+; RUN: llc -mtriple aarch64-linux-gnu   -mattr=+bti -mattr=+pauth              -asm-verbose=false \
+; RUN:   -o - %s | FileCheck %s --check-prefixes=ELF,CHECK
+; RUN: llc -mtriple aarch64-linux-gnu   -mattr=+bti -mattr=+pauth -global-isel -asm-verbose=false \
+; RUN:   -o - %s | FileCheck %s --check-prefixes=ELF,CHECK
+; RUN: llc -mtriple aarch64-linux-gnu   -mattr=+bti -mattr=+pauth -fast-isel   -asm-verbose=false \
+; RUN:   -o - %s | FileCheck %s --check-prefixes=ELF,CHECK
 
-; ptrauth tail-calls can only use x16/x17 with BTI.
+;; ptrauth tail-calls can only use x16/x17 with BTI.
```

Comment:
do we need this change? :)

Reply:
It looks like that new tests use a convention: for "special" lines like RUN, CHECK, etc, use ';' prefix, and for "actual comments" - use ';;' prefix. I've not found this in official guidelines, but it's clear from existing tests and I also feel that there were PRs where this was discussed.

If the verb in the question is **need**, then no, we don't need that, it's just a cosmetic change.